justfile

```diff
@@ -66,7 +66,7 @@ sign version team hd-path='':
     TEAM={{team}}
     just check-team $TEAM
 
-    HD_PATH={{hd-path}}
+    HD_PATH="{{hd_path}}"
```

Comment:
Code before & after change behaves exactly the same.
It would work with `hd-path` & `hd_path`